### PR TITLE
Hide dialpad when it is not a PSTN call

### PR DESCRIPTION
**Pull request by carocao-msft**
# What
Hide dialpad when it is not a PSTN call

# Why
Dialpad should not show up in non  PSTN call

# How Tested
start a call and click on more button, should not see dialpad

start a PSTN call and click on more button, should see dialpad

# Process & policy checklist

- [ ] I have updated the project documentation to reflect my changes if necessary.
- [ ] I have read the [CONTRIBUTING](https://github.com/Azure/communication-ui-library/blob/main/CONTRIBUTING.md) documentation.

**Is this a breaking change?**

- [ ] This change causes current functionality to break.

### CODE REVIEW

File: packages/react-composites/tests/browser/call/hermetic/DtmfDialpad.test.ts [hunk 1]
```diff
@@ -6,20 +6,34 @@ import { dataUiId, isTestProfileStableFlavor, pageClick, stableScreenshot, waitF
...
     const paul = defaultMockRemoteParticipant('Paul Bridges');
-    const vasily = defaultMockRemoteParticipant('Vasily Podkolzin');
 
-    const participants = [paul, vasily];
-    const initialState = defaultMockCallAdapterState(participants);
+    const participant = [paul];
+    const initialState = defaultMockCallAdapterState(participant);
 
+    //PSTN call has alternate caller id
+    initialState.alternateCallerId = '+1676568678999';
```
Comment (edwardlee-msft) [inline packages/react-composites/tests/browser/call/hermetic/DtmfDialpad.test.ts:18]: Is this a "random" number not associated with a real phone number?
I am unsure whether or not we have numbers we can use for our testing.

File: packages/react-composites/tests/app/callwithchat/LiveApp.tsx [hunk 2]
```diff
@@ -34,7 +34,9 @@ export function LiveApp(props: { queryArgs: QueryArgs }): JSX.Element {
       displayName: args.displayName,
       credential,
       endpoint: args.endpoint,
-      locator
+      locator,
+      /* @conditional-compile-remove(PSTN-calls) */
+      alternateCallerId: '+182927203720'
```
Comment (edwardlee-msft) [inline packages/react-composites/tests/app/callwithchat/LiveApp.tsx:39]: Same as my other question. Is this a number that is permitted to be used for testing?

File: change/@internal-react-composites-b482f6b7-6a84-44ef-b79f-e7cd47c9570c.json [hunk 3]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
+  "comment": "hide dialpad when it's not PSTN",
```
Comment (edwardlee-msft) [inline change/@internal-react-composites-b482f6b7-6a84-44ef-b79f-e7cd47c9570c.json:3]: nit:
Change to "Dialpad hidden in non PSTN 1:1 Calls."

Comment (carocao-msft) [reply change/@internal-react-composites-b482f6b7-6a84-44ef-b79f-e7cd47c9570c.json]: updated

File: packages/react-composites/tests/browser/call/hermetic/DtmfDialpad.test.ts (hunk 1 above)
Comment (edwardlee-msft) [reply packages/react-composites/tests/browser/call/hermetic/DtmfDialpad.test.ts:18]: As long as we ensure that the number will not be a valid number ever, I am okay with this.

**Review by anjulgarg — APPROVED** [review]

**Pull request merged**